extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -299,7 +302,12 @@ export class GmailElementReplacer implements WebmailElementReplacer {
         if (!convoUpperIcons.is('.appended') || convoUpperIcons.find('.use_secure_reply').length) {
           // either not appended, or appended icon is outdated (convo switched to encrypted)
           this.addfcConvoIcon(convoUpperIcons, this.factory.btnWithoutFc(), '.show_original_conversation', () => {
-            convoUpperIcons.find('.gZ').trigger('click');
+            const linkWithGmKeyId = $(this.sel.linkWithGmIdKey).attr('data-recorded-src') || '';
+            const dataMsgId = $(this.sel.dataMsgId).attr('data-message-id')?.replace('#', '') || '';
+            const tokenParam = new URLSearchParams(linkWithGmKeyId).get('token') || '';
+            const gmIdKey = JSON.parse(tokenParam)[1];
+            const webmailUrl = Google.webmailUrl(this.acctEmail) + `/?ik=${gmIdKey}&permmsgid=${dataMsgId}&view=lg`;
+            window.open(webmailUrl, '_blank', `height=680, width=620, top=30, left=${(screen.width - 620) / 2}`);
```

Comment:
It seems we overcomplicated this functionality - I checked previous code and there we just triggered click on the button which opened full thread in a separate window. And now we show only original of the last message.

So let's make it work as before - clicking `see original` should just trigger click on this button:

<img width="360" alt="Screenshot 2023-10-23 at 13 00 20" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/9a09d140-ca6a-41f7-bc69-57c33f90c264">

Reply:
Ok, the one I initially wrote requires too much. That's why we used `.trigger('click');` to call that one. I have fixed it and now working well :)